propel/resource_metric_test.go

```diff
@@ -169,3 +189,108 @@ func Test_expandMetricFilters(t *testing.T) {
 		})
 	}
 }
+
+func TestAccPropelMetricBasic(t *testing.T) {
+	ctx := map[string]interface{}{}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckPropelMetricDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckPropelMetricConfigBasic(ctx),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckPropelDataPoolExists("propel_metric.baz"),
+					resource.TestCheckResourceAttr("propel_metric.baz", "type", "CUSTOM"),
+					resource.TestCheckResourceAttr("propel_metric.baz", "expression", "SUM(quantity) / COUNT()"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckPropelMetricConfigBasic(ctx map[string]interface{}) string {
+	// language=hcl-terraform
+	return Nprintf(`
```

Comment:
This comment will tell the IDE that this terraform code inside a string, and we will be provided syntax highlighting and code completion

Reply:
cool